docs/model_objective/model-overview.md

```diff
@@ -4,7 +4,6 @@ Energy system models are used to measure the impact of climate policy and find p
 TIAM (TIMES Integrated Assessment Model), is a technology-rich bottom-up optimization model minimizing the total discounted system costs. On a spatial level, TIAM considers 16 regions, in which some regions represent countries (e.g., the US) and others are aggregated (e.g., Africa); see Figure 1. [Region description](../regional_coverage/region-description.md) provides further information about the TIAM regions. 
 
 ![TIAM Map](./figs/tiam-map.png) 
-
```

Comment:
For example a blank line here was introduced in the previous PR, but is being removed now.